guides/common/modules/proc_configuring-the-active-directory-authentication-source-on-projectserver.adoc

```diff
@@ -0,0 +1,58 @@
+= Configuring the Active Directory authentication source on {ProjectServer}
+
+Enable Active Directory (AD) users to access {Project} by configuring the corresponding authentication provider on your {ProjectServer}.
+
+.Prerequisites
+* The base system of your {ProjectServer} must be joined to an Active Directory (AD) domain.
+For more information, see link:https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/8/html-single/integrating_rhel_systems_directly_with_windows_active_directory/index#connecting-rhel-systems-directly-to-ad-using-sssd_integrating-rhel-systems-directly-with-active-directory[Connecting RHEL systems directly to AD using SSSD].
+
+.Procedure
+. Configure the System Security Services Daemon (SSSD) to use the AD access control provider to evaluate and enforce Group Policy Object (GPO) access control rules for the `foreman` PAM service:
+.. In your `/etc/sssd/sssd.conf` file, add the following lines to the `domain` section for the AD domain:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+[domain/_ad.example.com_]
+ad_gpo_access_control = enforcing
+ad_gpo_map_service = +foreman
+----
+ifndef::orcharhino[]
++
+For more information on GPOs, see link:{RHELDocsBaseURL}9/html/integrating_rhel_systems_directly_with_windows_active_directory/managing-direct-connections-to-ad_integrating-rhel-systems-directly-with-active-directory#how-sssd-interprets-gpo-access-control-rules_applying-group-policy-object-access-control-in-rhel[How SSSD interprets GPO access control rules] in _{RHEL}{nbsp}9 Integrating RHEL systems directly with Windows Active Directory_ or link:{RHELDocsBaseURL}8/html/integrating_rhel_systems_directly_with_windows_active_directory/managing-direct-connections-to-ad_integrating-rhel-systems-directly-with-active-directory#applying-group-policy-object-access-control-in-rhel_managing-direct-connections-to-ad[How SSSD interprets GPO access control rules] in _{RHEL}{nbsp}8 Integrating RHEL systems directly with Windows Active Directory_.
+endif::[]
+.. Restart SSSD:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# systemctl restart sssd
+----
+. Configure AD as the authentication source for your {ProjectServer}:
+.. Create an Apache keytab file if it does not already exist on your system:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# touch /etc/httpd/conf/http.keytab
+----
++
+You can leave the file empty.
+The `{foreman-installer}` command fails to configure the authentication source if the file is missing.
+.. Enable IPA authentication:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# {foreman-installer} --foreman-ipa-authentication=true
+----
+.. Restart Apache:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# systemctl restart httpd
+----
+
+.Verification
+* Log in to {ProjectwebUI} by entering the credentials of a user defined in AD.
+Enter the user name and the domain name, for example: _ad_user_@_ad.example.com_.
```

Comment:
If there are no separate fields for user name and domain name, then I recommend:

Enter the user name in user principal name (UPN) format, for example _ad_user_@_ad.example.com_.

Reply:
You can mention here that this requires some specific browser settings.

With respect to "[5.4.2. Kerberos configuration in web browsers](https://theforeman-foreman-documentation-preview-pr-3149.surge.sh/nightly/Installing_Server/index-satellite.html#Kerberos_Configuration_in_Web_Browsers_satellite)": [IE is EOL](https://learn.microsoft.com/en-us/lifecycle/products/internet-explorer-11). I would mention Firefox, Chrome, an Edge.

Reply:
> If there are no separate fields for user name and domain name, then I recommend:
> 
> Enter the user name in user principal name (UPN) format, for example _ad_user_@_ad.example.com_.

This is great, I wasn't aware of the term UPN. Thanks!

Reply:
> You can mention here that this requires some specific browser settings.

You mean being able to log in with just the user name? That's already in the works, I hope to submit another PR once I finalize this one. (More changes will be needed to cover that and I don't want this PR to grow out of scope... again.)

> With respect to "[5.4.2. Kerberos configuration in web browsers](https://theforeman-foreman-documentation-preview-pr-3149.surge.sh/nightly/Installing_Server/index-satellite.html#Kerberos_Configuration_in_Web_Browsers_satellite)": [IE is EOL](https://learn.microsoft.com/en-us/lifecycle/products/internet-explorer-11). I would mention Firefox, Chrome, an Edge.

O-oh. I.. think we will need to take a look at the whole section then. If it's that old, we should review it more thoroughly. Thanks again!